concepticondata/conceptlists/Hartmann-2013-162.tsv

```diff
@@ -0,0 +1,163 @@
+Hartmann-2013-162-36	36	COME	1446	COME	The boy came (to the village).	A comes (to L)	New meaning	come
+Hartmann-2013-162-37	37	COME (1)			The boy came.	S comes	Old meaning	come-1
+Hartmann-2013-162-38	38	COME (2)			The boy came to/from the village.	A comes (from/to L)	New meaning	come-2
+Hartmann-2013-162-39	39	COOK	1100	COOK (SOMETHING)	The women cooked the meat.	A cooks P	Extended list	cook
+Hartmann-2013-162-40	40	COUGH	879	COUGH	The old man coughed.	S coughs	Core list	cough
+Hartmann-2013-162-41	41	COUGH AT			The man choughed at my face.	A coughs at P	New meaning	cough-at
+Hartmann-2013-162-42	42	COVER	761	COVER	The woman covered the boy with a blanket.	A covers P (with X)	Core list	cover
+Hartmann-2013-162-43	43	CRAM			The boy crammed peanuts into his mouth.	X crams Y into Z	New meaning	cram
+Hartmann-2013-162-44	44	CROSS			The boys crossed the road.	X crosses Y	New meaning	cross
+Hartmann-2013-162-45	45	CRY	1839	CRY	The baby is crying.	S cries	Old meaning	cry
+Hartmann-2013-162-46	46	CRY ABOUT			The woman cries about her loss.	A cries about X	New meaning	cry-about
+Hartmann-2013-162-47	47	CUT	1432	CUT	The woman cut the bread with a sharp knife.	A cuts P (with I)	Core list	cut
+Hartmann-2013-162-48	48	DESERVE			The woman deserves credit.	A deserves P	New meaning	deserve
+Hartmann-2013-162-49	49	DIE	1494	DIE	The snake died.	S dies	Core list	die
+Hartmann-2013-162-50	50	DIG	1418	DIG	The woman is digging for potatoes.	A digs (for X)	Extended list	dig
+Hartmann-2013-162-51	51	DO	813	DO	The man did his work.	X does Y	New meaning	do
+Hartmann-2013-162-52	52	DREAM	2455	DREAMING OR DREAM	The woman was dreaming.	E dreams	New meaning	dream
+Hartmann-2013-162-53	53	DRESS			The mother dressed her daughter	A dresses P	Core list	dress
+Hartmann-2013-162-54	54	DRINK	1401	DRINK	The man drank strong coffee.	X drinks Y	New meaning	drink
+Hartmann-2013-162-55	55	DRIVE OUT			The man drove out his boat.	A drives P out	New meaning	drive-out
+Hartmann-2013-162-56	56	DROP	2866	DROP (SOMETHING)	The girl dropped the cup.	A drops P	New meaning	drop
+Hartmann-2013-162-57	57	DROWN	1904	DROWN	The man drowned in a river.	S drowns	New meaning	drown
+Hartmann-2013-162-58	58	DRY	1786	DRY UP	The sun has dried out the ground.	A dries P	New meaning	dry
+Hartmann-2013-162-59	59	EAT	1336	EAT	The boy ate the fruit.	A eats P	Core list	eat
+Hartmann-2013-162-60	60	ENTER	749	ENTER	The man entered (the house).	X enters somewhere	New meaning	enter
+Hartmann-2013-162-61	61	FALL	1280	FALL	The horse fell.	S falls	Old meaning	fall
+Hartmann-2013-162-62	62	FEAR	1419	FEAR (BE AFRAID)	The man feared the bear.	E fears M	Core list	fear
+Hartmann-2013-162-63	63	FEEL COLD			I’m cold.	S is cold	Core list	feel-cold
+Hartmann-2013-162-64	64	FEEL PAIN	2561	HURT (SENSE PAIN)	My arm is hurting. =I’m feeling pain in my arm.	E feels pain in M	Core list	feel-pain
+Hartmann-2013-162-65	65	FEEL SICK			The boy felt sick.	E feels sick	New meaning	feel-sick
```

Comment:
[Bulakh-2013-870](https://concepticon.clld.org/values/Bulakh-2013-870-664) "to feel sick" is mapped to SICK.

Reply:
I already mapped SICK in this list (to BE ILL). Should this be changed?

Reply:
Ah ok, tricky. But I'd suggest that it is more consistent with the other mappings if we leave this unmapped and map BE ILL to SICK. So nothing to do here.